README.md

```diff
@@ -1,6 +1,5 @@
 # Conduit Connector Kafka
 ![scarf pixel](https://static.scarf.sh/a.png?x-pxid=713ea3ba-66e0-4130-bdd0-d7db4b8706a0)
-
```

Comment:
If anything, I would _add_ another line 😄 We have a markdown linter in the [conduit repo](https://github.com/DavidAnson/markdownlint-cli2-action) which would complain in this case if I'm not mistaken. I can't find where these rules come from, but iirc the best practice is to leave an empty line between chapters and paragraphs.

Reply:
Ah, here they are https://github.com/DavidAnson/markdownlint/blob/v0.32.1/doc/Rules.md, MD022

Reply:
Oh I just noticed the empty line that shows up on the main page, that's probably why you want to add this.

What if we move the pixel to the bottom of the page? 🤔

Reply:
@lovromazgon that's an option. The idea to keep it on top was to make sure that it wasn't removed accidentally if refactoring the document. I'll close this PR for now as it's not that important.